cli/lib/info.js

```diff
@@ -5,8 +5,9 @@
  * See the LICENSE file for more information.
  */
 
-const
-	appc = require('node-appc'),
```

Comment:
+1 On not declaring each variable deceleration being separate, as always [there's a rule for that](http://eslint.org/docs/rules/one-var) but not sure whether all teams would buy in to having that in the config. There's also a rule for [strict mode](http://eslint.org/docs/rules/strict) which imo if our min ver will be node 4 should be added to the config (again doubt all times would buy in though)